src/mono/browser/runtime/cwraps.ts

```diff
@@ -130,9 +130,9 @@ const fn_signatures: SigLine[] = [
     [true, "mono_jiterp_end_catch", "void", []],
     [true, "mono_interp_pgo_load_table", "number", ["number", "number"]],
     [true, "mono_interp_pgo_save_table", "number", ["number", "number"]],
-    [() => !runtimeHelpers.emscriptenBuildOptions.enablePerfTracing, "mono_jiterp_prof_enter", "void", ["number", "number"]],
-    [() => !runtimeHelpers.emscriptenBuildOptions.enablePerfTracing, "mono_jiterp_prof_samplepoint", "void", ["number", "number"]],
-    [() => !runtimeHelpers.emscriptenBuildOptions.enablePerfTracing, "mono_jiterp_prof_leave", "void", ["number", "number"]],
+    [() => !runtimeHelpers.emscriptenBuildOptions.enablePerfTracing || runtimeHelpers.emscriptenBuildOptions.enableBrowserProfiler, "mono_jiterp_prof_enter", "void", ["number", "number"]],
```

Comment:
In these 3 lines we are negating `enablePerfTracing` to register jiterpreter's methods (register if profiler is off) but then, in src/mono/browser/runtime/jiterpreter.ts we don't use that negation. It's not this PR's change but I don't follow

Reply:
It's lazy loading, not skipping the method. So we are lazy loading if `enablePerfTracing` is off but `enableBrowserProfiler` on? Why?

Reply:
this is typo, many thanks!